doc/lua_api.txt

```diff
@@ -3063,27 +3101,33 @@ callback, `nil` is returned.
 
 The following Mapgen objects are currently available:
 
-### `voxelmanip`
+`voxelmanip`
```

Comment:
This is a list of mapgen objects, not section titles, best leave these 6 unchanged or change to a list.